learn2learn/algorithms/maml.py

```diff
@@ -103,14 +107,32 @@ def __init__(self,
             allow_unused = allow_nograd
         self.allow_unused = allow_unused
 
+    def _init_lslr_parameters(
+        self, model: torch.nn.Module, adaptation_steps: int, init_lr: float
+    ) -> torch.nn.ParameterDict:
+        lslr = torch.nn.ParameterDict()
+        for layer_name, layer in model.named_modules():
+            # If the layer has learnable parameters
+            if (
+                len(
+                    [
+                        name
+                        for name, param in layer.named_parameters(recurse=False)
+                        if param.requires_grad
+                    ]
+                )
+                > 0
+            ):
+                lslr[layer_name.replace(".", "-")] = torch.nn.Parameter(
+                    data=torch.ones(adaptation_steps) * init_lr,
+                    requires_grad=True,
+                )
+        return lslr
```

Comment:
@DubiousCactus this should be removed, right? In general, can you stash this file so it keeps the original formatting?

Reply:
Yes, sorry about that.